extractor/src/main/java/org/schabi/newpipe/extractor/stream/SubtitlesStream.java

```diff
@@ -276,6 +299,19 @@ public boolean isAutoGenerated() {
         return autoGenerated;
     }
 
+    /**
+     * Whether the subtitles are translated automatically by a machine.
+     * <p>
+     * Some streaming services provide automatically translated subtitles.
+     * YouTube, for example, uses Google translator to generate translated subtitles.
+     * Automatically translated subtitles might not coincide completely with the original text.
+     * </p>
+     * @return {code true} if the subtitles are auto-translated, {@link false} otherwise
```

Comment:
```suggestion
     *
     * <p>
     * Some streaming services provide automatically translated subtitles.
     * YouTube, for example, uses Google Translator to generate translated subtitles.
     * Automatically translated subtitles might not coincide completely with the original text.
     * </p>
     *
     * @return {code true} if the subtitles are auto-translated, {@link false} otherwise
```